packages/pricing/src/models/price-rule.ts

```diff
@@ -39,9 +39,6 @@ export default class PriceRule {
   })
   rule_type: RuleType
 
-  @Property({ columnType: "boolean", default: false })
-  is_dynamic: boolean
-
```

Comment:
question: should this require an additional migration? or do we want to keep the db column but remove the property here?

Reply:
lets keep the migration there for now, we can always remove it at a later point.